### PR TITLE
chore: cleanup activity list

### LINKED ISSUE #819: ️🛠 - Cleanup activity list
### Task description

- [x] Dash at consolidation activities
- [x] Make searchbar not toggleable
- [x] Fix expiration time input style

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
     import { localize } from '@core/i18n'
     import { nftSearchTerm, ownedNfts, queriedNfts } from '@core/nfts/stores'
-    import { FontWeight, Illustration, NftGallery, ReceiveButton, SearchInput, Text } from '@ui'
+    import { FontWeight, Illustration, NftGallery, ReceiveButton, Text } from '@ui'
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte]: Let's use the bloom-ui Text component

File: packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte [hunk 2]
```diff
@@ -75,11 +75,13 @@
...
-                bind:selected={selectedExpirationPeriod}
-                disabled={disableChangeExpiration || disableAll}
-            />
+            <div>
+                <DateTimePickerMenu
+                    bind:value={expirationDate}
+                    bind:selected={selectedExpirationPeriod}
+                    disabled={disableChangeExpiration || disableAll}
+                />
+            </div>
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte:84]: What's the benefit of these `div`s?

File: packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte [hunk 3]
```diff
@@ -94,11 +96,14 @@
...
-                disabled={disableChangeTimelock || disableAll}
-            />
+            <div>
+                <DateTimePickerMenu
+                    bind:value={timelockDate}
+                    bind:selected={selectedTimelockPeriod}
+                    disabled={disableChangeTimelock || disableAll}
+                />
+                <div></div>
+            </div>
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte]: What's the benefit of these `div`s?

**Review by Tuditi — CHANGES_REQUESTED** [review]
I requested some changes and let's move this into draft mode until we sorted out the height of the search input.

It's too large as discussed on today's stand-up.
image

File: packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte (hunk 2 above)
Comment (MarkNerdi) [reply packages/desktop/views/dashboard/send-flow/views/components/StardustTransactionDetails.svelte:84]: this div ensures that it is sized accordingly

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**